docs/en/ufs/Minio.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: global
-title: Minio
-nickname: Minio
+title: MinIO
+nickname: MinIO
 group: Under Stores
 priority: 10
 ---
 
 * Table of Contents
 {:toc}
 
-This guide describes how to configure Alluxio with [Minio](https://minio.io/) as the
+This guide describes how to configure Alluxio with [MinIO](https://minio.io/) as the
```

Comment:
I think their new website is `https://min.io/`?

Reply:
It redirects to min.io but I'll update the url